src/Tutor.Core/DomainModel/KnowledgeComponents/AssessmentEventSelector.cs

```diff
@@ -1,19 +1,16 @@
-using FluentResults;
 using System.Collections.Generic;
 using System.Linq;
+using FluentResults;
 using Tutor.Core.DomainModel.AssessmentEvents;
-using Tutor.Core.DomainModel.KnowledgeComponents;
 
-namespace Tutor.Core.InstructorModel.Instructors
+namespace Tutor.Core.DomainModel.KnowledgeComponents
 {
-    public class DefaultInstructor : IInstructor
+    public class AssessmentEventSelector : IAssessmentEventSelector
```

Comment:
AESelector is generic and this class is not. LeastCorrectAssessmentEventSelector?